projects/aca-content/viewer/src/lib/components/viewer/viewer.component.ts

```diff
@@ -197,7 +197,10 @@ export class AcaViewerComponent implements OnInit, OnDestroy {
     this.actions$
       .pipe(ofType<RefreshPreviewAction>(ViewerActionTypes.RefreshPreview), takeUntil(this.onDestroy$))
       .subscribe((action: RefreshPreviewAction) => {
-        this.displayNode(action?.payload?.entry?.id);
+        if (action.node instanceof Node) {
```

Comment:
instead of instanceof could you please just update 'node' type inside RefreshPreviewAction to Node so then no need to check this type here

Reply:
I did it this way, because I get this error and I am not sure how to fix it, model also contains this field as a Node

<img width="1301" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/156320606/5f5769f8-f3f0-48b8-95ca-fb4a8c306f65">

<img width="416" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/156320606/c089f90f-9d62-498b-854a-91bd4cbf0ca5">

Reply:
ok, I found the issue, in actions it didn't import Node from api, but the weird thing is that also it didn't show any error or request for import 🤷‍♀️

Reply:
@tamaragruszka the reason why it does not show any request for import when you removed that import was because there is other Node inside ts library for some DOM representation 
<img width="1037" alt="image" src="https://github.com/Alfresco/alfresco-content-app/assets/115619721/3a47ac88-ba01-4c8b-ba0e-8d62a6074797">